.github/workflows/ansible-test.yml

```diff
@@ -1,17 +1,33 @@
 # README FIRST
-# 1. If you don't have unit tests, remove that section.
-# 2. If your collection depends on other collections ensure they are installed,
+# 1. Subscribe to https://github.com/ansible-collections/news-for-maintainers
+#    (click the Watch button on the homepage > Custom > Issues)
+#    and keep this matrix up to date in accordance to related announcements.
+#    Timely add new ansible-core versions and consider dropping support
+#    and testing against its EOL versions.
+# 2. If your collection repository is under the ansible-collections org,
+#    please keep in mind that the number of GHA jobs is limited
+#    and shared across all the collections in the org.
+#    So, focusing on good test coverage of your collection,
+#    please avoid testing against unnecessary entities such as
+#    ansible-core EOL versions your collection does not support.
```

Comment:
```suggestion
#    ansible-core EOL versions or versions your collection does not support.
```

Avoid testing against ansible-core versions that are _not_ EOL yet but not supported by the collection helps, too.